checks.d/mysql.py

```diff
@@ -863,12 +863,17 @@ def _get_replica_stats(self, db):
             self.warning("Privileges error getting replication status (must grant REPLICATION CLIENT): %s" % str(e))
             return {}
 
-    def _get_slave_status(self, db, nonblocking=True):
+    def _get_slave_status(self, db, above_560):
```

Comment:
Not sure what's the benefits in doing that. `_get_slave_status` is only called here and once.